app/models/checklist_template.rb

```diff
@@ -9,4 +9,5 @@
 #  updated_at  :datetime         not null
 #
 class ChecklistTemplate < ApplicationRecord
+  has_many :items, class_name: "ChecklistTemplateItem", dependent: :destroy
```

Comment:
minor but when comparing to class names, I like to do `ChecklistTemplateItem.name` because it guarantees the class exists and is loadable before getting the string name out of it. It's technically less efficient than a frozen string but I think is a bit more defensive against typos. I have heard arguments against this too though, so not a big deal either way.